app/_data/docs_nav_kuma_dev.yml

```diff
@@ -19,13 +19,95 @@ items:
             url: "/introduction/how-kuma-works/#vm-and-k8s-support"
           - text: Kuma vs XYZ
             url: "/introduction/how-kuma-works/#kuma-vs-xyz"
-      - text: Deployments
-        url: /introduction/deployments/
       - text: Enterprise
         url: /introduction/enterprise/
       - text: Changelog
         absolute_url: true
         url: https://github.com/kumahq/kuma/blob/master/CHANGELOG.md
+  - title: Kuma in Production
+    group: true
+    items:
+      - title: Deployment topologies
+        group: true
+        items:
+          - text: Overview
+            url: /production/deployment/
+          - text: Standalone deployment
+            url: /production/deployment/stand-alone/
+          - text: Multi-zone deployment
+            url: /production/deployment/multi-zone/
+          - text: Default ports
+            url: /production/deployment/networking/
+      - title: Control plane deployment
+        group: true
+        items:
+          # - text: Kong Mesh license
+            # url: /production/
+          - text: Deploy a multi-zone global control plane
+            url: /production/cp-deployment/multi-zone/ 
+            group: true
+            items:
+              # - text: Add a zone
+                # url: /production/
+              # - text: Configure multi-zone authentication
+                # url: /production/
+              - text: Zone Ingress
+                url: /production/cp-deployment/zone-ingress/
+              - text: Zone Egress
+                url: /production/cp-deployment/zoneegress/
+              - text: Configure zone proxy authentication
+                url: /production/cp-deployment/zoneproxy-auth/
+          - text: Deploy a standalone control plane
+            url: /production/stand-alone/
+          - text: Control plane configuration reference
+            url: /production/cp-config-reference/
+            src: /generated/kuma-cp/
```

Comment:
@fabianrbz I don't think I formatted the src of this correctly because it won't display.

Reply:
I think the problem with this one is that you set `generate: false` so it won't generate the page for that item, that should be ok but the thing is that the URL doesn't match the file so it will return a 404.

However, if you remove that, it fails because it will try to find `/generated/kuma-cp` under `app/_src` which isn't there.

I guess we have two options here: make the URL match the file path or add support for using files outside of `app/_src` in this case `app/docs/` as a fallback in case it doesn't find them.

Reply:
> make the URL match the file path

I was going to pick this option, but it looks like this is a generated file and I don't think it's listed in the nav in the current docs. The URL for this in the public docs is `/generated/kuma-cp/`, but that doesn't seem to work and I don't know what the actual file path is.

Reply:
something like this should work:
```
- text: Control plane configuration reference
   url: /generated/kuma-cp/
   generate: false
```